Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -1,13 +1,26 @@
 module Superenv
+  alias x11? x11
+
   # @private
   def self.bin
     (HOMEBREW_SHIMS_PATH/"linux/super").realpath
   end
 
+  def xorg_recursive_deps
+    if xorg_installed?
```

Comment:
Minor style point, but I prefer:
```ruby
def xorg_recursive_deps
  return [] unless xorg_installed?
  @xorg_deps ||= Formula["linuxbrew/xorg/xorg"].recursive_dependencies.map(&:to_formula)
rescue FormulaUnavailableError
  []
end
```
I was going to say that the `rescue` clause is unnecessary because of the first `xorg_installed?` guard, but it's possible that the user installs `xorg` and then untaps `xorg`.